saleor/order/models.py

```diff
@@ -64,6 +64,12 @@ class Order(models.Model, ItemSet):
                                              editable=False)
     token = models.CharField(
         pgettext_lazy('Order field', 'token'), max_length=36, unique=True)
+    currency = models.CharField(
+        pgettext_lazy('Order field', 'currency'),
+        max_length=3, default=settings.DEFAULT_CURRENCY)
+    rate = models.DecimalField(
```

Comment:
We also should know order currency, actually wombat requires currency for order serialization.
With single `total` field as a `PriceField` with currency set to store's default currency, we will lose information about currency rate used to convert the price